docs/index.md

```diff
@@ -144,37 +144,49 @@ npm run kakunin
 In order to make cucumber steps autosuggestion work properly in JetBrains tools, make sure your project is `ECMAScript 6` compatible and you have `cucumberjs` plugin installed.
 Due to non-resolved issue in Jetbrains editors ([see here](https://youtrack.jetbrains.com/issue/WEB-11505)) we'll have to do one more step:
  
-Go to `step_definitions` directory 
+Go to `step_definitions` directory
 ```bash
-cd step_definitions
+cd step_definitions/web
```

Comment:
There is no web, res or common directory inside step_definitions (in our project). This should be a single directory without subdirectories.